Py/DICOMSegmentationPlugin.py

```diff
@@ -143,54 +143,67 @@ def load(self,loadable):
       (success,labelNode) = slicer.util.loadLabelVolume(labelFileName, returnNode=True)
 
       # Initialize color and terminology from .info file
-      # Format of the .info file:
+      # See SEG2NRRD.cxx for how it's written.
+      # Format of the .info file (no leading spaces, modifiers are optionally written if present):

```

Comment:
The reason I merged is that I don't want to spend time refining all the wrinkles in the first iteration. "Release often release early". Error checking will need to be refined anyway at different levels, including SEG2NRRD.